view/theme/frio/img/dynamic.php

```diff
@@ -7,7 +7,7 @@
       'dsn' => 'mysql:host=127.0.0.1;dbname=installer',
       'username' => 'root',
       'password' => 'root',
-      'charset' => 'utf8',
+      'charset' => 'utf8mb4',
```

Comment:
@annando Here you are https://github.com/friendica/friendica/pull/3326/commits/6cc4f8b0a9673c77320905f4ae18fdacb76b8f9f